.github/workflows/release.yml

```diff
@@ -36,3 +36,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+
+      - name: setup Node to publish to GitHub Packages
+        uses: actions/setup-node@v1
+        with:
+          registry-url: 'https://npm.pkg.github.com'
+          node-version: '12.x'
+
+      - name: publish to GitHub Packages
+        run: yarn publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
It is correct changes.
Please read the official documents.
https://docs.github.com/en/packages/quickstart#publishing-your-package

Reply:
Oops. 
It's my misunderstood as you said.
Thanks for the info.